shared/models/videos/video-transcoding.model.ts

```diff
@@ -12,7 +12,9 @@ export type EncoderOptionsBuilder = (params: {
 export interface EncoderOptions {
   copy?: boolean // Copy stream? Default to false
 
-  outputOptions: string[]
+  inputOptions?: string[]
+  videoFilters?: string[]
```

Comment:
Please remove this, to not couple too much with `ffmpeg-fluent`. We can already use video filters using `outputOptions`

Reply:
I agree that less coupling with `ffmeg-fluent` is better.
However, specifying filters directly in `outputOptions` entails that there is no use of `scale()` or `videoFilters()` anywhere else in peertube code.
`ffmpeg-fluent` keeps track of all filters and generate a single `-filter:v` argument with the entire filter chain because `ffmpeg` will only use the latest `-filter:v` argument and ignore previous ones.

To illustrate the issue:
```js
command .input('in.mkv')
        .size('?x360')
        .outputOptions('-vf vflip')
        .output('out.mp4')
        .on('start', (cmd) => {
            console.log(cmd)
        })
        .run()
```

Generates this command line `ffmpeg -i in.mkv -y -filter:v scale=w=trunc(oh*a/2)*2:h=360 -vf vflip out.mp4` which will ignore the scale filter and only apply the vflip.